zstacklib/zstacklib/storage/lvm/pv.py

```
@@ -0,0 +1,273 @@
+    
+    Args:
+        vg_uuid: Volume group UUID/name
+        disk_path: Device path to add
+        metadata_size: Metadata area size (e.g., "16M")
+        
+    Raises:
+        Exception: If the disk was not added successfully
+    """
+    bash.bash_errorout(f"vgextend --metadatasize {metadata_size} {vg_uuid} {disk_path}")
+    if bash.bash_r(f"pvs --nolocking -t --readonly {disk_path} | grep {vg_uuid}"):
+        raise Exception(f"disk {disk_path} not added to vg {vg_uuid} after vgextend")
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)